libvmi/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set(libvmi_src
     arch/arm_aarch32.c
     arch/arm_aarch64.c
     driver/driver_interface.c
+    driver/driver_utils.c
```

Comment:
I think this would be more properly abstracted into the `arch` interface given the EPT settings are architecture-specific, not hypervisor specific.